balances-adjustment/src/test/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationSimpleDcTest.java

```diff
@@ -338,4 +313,53 @@ void testUnBalancedNetworkReport() throws IOException {
         BalanceComputationAssert.assertReportEquals("/unbalancedNetworkReport.txt", reporter);
     }
 
+    private abstract class AbstractLoadFlowProviderMock implements LoadFlowProvider {
```

Comment:
you could extend `AbstractNoSpecificParametersLoadFlowProvider` to cut down the length of that class. I think then you only need getName and getVersion